app/services/user.ts

```diff
@@ -144,7 +148,13 @@ export class UserService extends PersistentStatefulService<IUserServiceState> {
         return res.text();
       })
       .then(valid => {
-        if (valid.match(/false/)) this.LOGOUT();
+        if (valid.match(/false/)) {
+          this.LOGOUT();
```

Comment:
So this API always returns true now but we may change that.  Can be a separate PR, but I think we should display a message that says something like "You have been logged out.  Please log back in to continue using Streamlabs OBS" and then take them to the login screen.

Reply:
Added a message